modules/is-restorable/.nycrc

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I think `.nycrc` should be placed in root directory.
And replace package.json test script with `--nycrc-path` option.